### PR TITLE
feat: Added features for square root,modulo and exponent

### PR DESCRIPTION
**Pull request by Pritom14**
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [ ] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Review by mayank-kgp — COMMENTED** [review]
Please don't send changes in any files in ./idea.

**Comment by Pritom14** [comment]
Okay, just a moment

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mayank-kgp) [inline .idea/vcs.xml:1]: Please don't send any files in .idea/ directory.

**Review by mayank-kgp — COMMENTED** [review]
Changes Requested.

File: app/src/main/java/com/example/mayankjindal/voicecalculator/MainActivity.java [hunk 2]
```diff
@@ -31,7 +31,10 @@
     // IDs of all the numeric buttons
     private int[] numericButtons = {R.id.btnZero, R.id.btnOne, R.id.btnTwo, R.id.btnThree, R.id.btnFour, R.id.btnFive, R.id.btnSix, R.id.btnSeven, R.id.btnEight, R.id.btnNine};
     // IDs of all the operator buttons
-    private int[] operatorButtons = {R.id.btnAdd, R.id.btnSubtract, R.id.btnMultiply, R.id.btnDivide};
+    private int[] operatorButtons = {R.id.btnAdd, R.id.btnSubtract, R.id.btnMultiply, R.id.btnDivide, R.id.btnMod, R.id.btnExpo};
+
```
Comment (mayank-kgp) [inline app/src/main/java/com/example/mayankjindal/voicecalculator/MainActivity.java:35]: Why this blank line??

File: app/src/main/java/com/example/mayankjindal/voicecalculator/MainActivity.java [hunk 3]
```diff
@@ -79,6 +85,21 @@ public void onClick(View v) {
...
+                if (lastNumeric && !stateError) {
+                    // Read the expression
+                    String txt = txtScreen.getText().toString();
+                    double x=Double.parseDouble(txt);
+                    double res=Math.sqrt(x);
+                    txtScreen.setText(Double.toString(res));
+                }
+            }
+        });
+
```
Comment (mayank-kgp) [inline app/src/main/java/com/example/mayankjindal/voicecalculator/MainActivity.java:102]: Why blank line?

**Review by mayank-kgp — COMMENTED** [review]
Changes requested

**Pull request closed without merging**